src/components/Stack/Stack.scss

```diff
@@ -105,6 +105,7 @@
 
 .Item {
   flex: 0 1 auto;
+  min-width: 0;
```

Comment:
When the stack width is smaller than the contents, the contents overflow each other. Maybe we could make them wrap to the next line? Not sure if this would introduce even more edge cases though

Reply:
Because this is an abstraction of Flexbox we might just want to leave it as is. Wrapping is left up to the props which should generate the right flexbox rules.